packages/mocaccino-dracut/build.yaml

```diff
@@ -0,0 +1,14 @@
+image: "quay.io/mocaccino/desktop:master"
+
+package_dir: /mocaccino-dracut
+
+prelude:
+- luet install sys-kernel/dracut
+
+steps:
+- | 
+   PACKAGE_VERSION=${PACKAGE_VERSION%\+*} && \
+   mkdir -p /${PACKAGE_NAME}/usr/bin && \
+   cp -rfv mocaccino-dracut  /${PACKAGE_NAME}/usr/bin/ && \
+   rm -rf /${PACKAGE_NAME}/usr/bin/*.dockerfile && \
```

Comment:
Did you ended up with a Docker file inside? I think you can drop it, in the keybase-tools it's there because I was copying all the scripts that had a suffix

Reply:
No, I was mimicking the keybase-tools. I figured you were doing something with docker that I didn't understand so I left it. Thanks for the info, I'll remove the line deleting the docker file.